nss-day-cohort-39/tabloid-full-stack-coulees
# Create, Edit, Delete Posts

### Changes
1. Created Add Post Form and related functions
2. Created Edit Post Form and related functions
3. Created Delete post button that initiates a confirmation modal and deletes upon user confirmation

### To Test
1. Create a post. Try leaving the publish date blank. (NOTE: for now all categoryIds for added posts are 1, until we can implement a select dropdown using the category repo/provider)
2. Delete a post written by your current user
3. Try editing a post written by the current user.
4. Ensure all the modals work properly.
5. Ensure merge conflicts were resolved correctly and Tags components are still functional
